tests/local_cluster.rs

```diff
@@ -38,3 +40,64 @@ fn test_spend_and_verify_all_nodes_3() -> () {
         num_nodes,
     );
 }
+
+#[test]
+#[ignore]
+fn test_rotate_nodes_3() -> () {
+    solana_logger::setup();
+    let num_nodes = 3;
+    let local = LocalCluster::new(num_nodes, 10_000, 100);
+    cluster_tests::rotate_leader_through_all_nodes(
+        &local.entry_point_info,
+        num_nodes,
+        Duration::from_secs(num_nodes as u64 * 60),
+    );
+
+    // Test that transactions can go through
+    cluster_tests::spend_and_verify_all_nodes(
+        &local.entry_point_info,
+        &local.funding_keypair,
+        num_nodes,
+    );
+}
+
+#[test]
+#[ignore]
+fn test_rotate_nodes_remove_leader_3() -> () {
+    solana_logger::setup();
+    let num_nodes = 3;
+    let local = LocalCluster::new(num_nodes, 10_000, 100);
+
+    // Check that all nodes cycle through leader role
+    cluster_tests::rotate_leader_through_all_nodes(
+        &local.entry_point_info,
+        num_nodes,
+        Duration::from_secs(num_nodes as u64 * 60),
+    );
+
+    // Exit the leader node
```

Comment:
Can you add an rpc to exit instead?  I’d like the tests to run against s testnet